Adding new topic UWP (#579)
github/explore: topics/uwp/index.md

---
aliases: universal-windows-platform
created_by: Microsoft

> display_name: UWP

Reviewer: ```suggestion
display_name: Universal Windows Platform
```

> short_description: UWP is an API that allows a developer to create apps that can run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens.

Reviewer: Would maybe 

```suggestion
short_description: An API for creating apps that run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens.
```

topic: uwp
wikipedia_url: https://en.wikipedia.org/wiki/Universal_Windows_Platform
---

> Universal Windows Platform(UWP) is an API that allows a developer to create apps that can run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens without having to create induvidual apps for each platform.

Reviewer: ```suggestion
Universal Windows Platform (UWP) is an API that allows developers to create cross-platform applications that are able to run on Windows 10, Windows 10 Mobile, Xbox One and HoloLens.
```
